changelog/18596.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+openapi: Add openapi response definitions to PKI path_fetch_*.go
```

Comment:
One comment is can we have a single changelog entry for all the response enhancements, rather than per file?

Reply:
Sure thing! Thanks for pointing that out.